[Bug] GitVersion.MSBuild fails when running "dotnet.exe pack"

**Describe the bug**
Running "dotnet.exe pack" fails with  `error MSB4044: The "GenerateGitVersionInformation" task was not given a value for the required parameter "Language"`.

## Expected Behavior
"dotnet.exe pack" running to completion.

## Actual Behavior
Now that #2502, #2518 and #2519 have been fixed in 5.6.2 my "dotnet.exe build" and "dotnet.exe test" jobs are passing.
Running "dotnet.exe pack" and "dotnet.exe pack --no-build" however both fail with `error MSB4044: The "GenerateGitVersionInformation" task was not given a value for the required parameter "Language"`.

## Context
It is no longer possible to build a NuGet package using "dotnet.exe pack".

## Your Environment

- GitVersion.MsBuild 5.6.2
- Windows 10

@arturcic sure, see https://github.com/dieterv/GitVersion-issues-2523
This is what I get when running  `powershell -File build.ps1`:
```
<snip>
========================================
Package
========================================
Executing task: Package
<snip>
C:\Program Files\dotnet\sdk\5.0.101\MSBuild.dll -maxcpucount -property:NoBuild=true -property:IncludeSymbols=true -property:Configuration=Release -target:pack -verbosity:m /bl:C:/Development/github.com/GitTools/GitVersion-issues-2523/build/artifacts/dotnet_pack.binlog C:/Development/github.com/GitTools/GitVersion-issues-2523/Reproducer.sln
C:\Users\dieter\.nuget\packages\gitversion.msbuild\5.6.3\tools\GitVersion.MsBuild.targets(110,9): error MSB4044: The "GenerateGitVersionInformation" task was not given a value for the required parameter "Language". [C:\Development\github.com\GitTools\GitVersion-issues-2523\source\Reproducer\Reproducer.csproj]
An error occurred when executing task 'Package'.
<snip>
```

--- a/src/GitVersion.MsBuild/msbuild/tools/GitVersion.MsBuild.props
+++ b/src/GitVersion.MsBuild/msbuild/tools/GitVersion.MsBuild.props
@@ -4,6 +4,7 @@
     <PropertyGroup>
         <GitVersionOutputFile>$(BaseIntermediateOutputPath)/gitversion.json</GitVersionOutputFile>
 
+        <Language Condition=" '$(Language)' == '' ">C#</Language>
         <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">$(MSBuildProjectDirectory)/../</SolutionDir>
         <GitVersionPath Condition="'$(GitVersionPath)' == '' And '$(GitVersionUseSolutionDir)' == 'true'">$(SolutionDir)</GitVersionPath>
         <GitVersionPath Condition="'$(GitVersionPath)' == ''">$(MSBuildProjectDirectory)</GitVersionPath>
@@ -62,6 +63,7 @@
         <UpdateVersionProperties Condition=" '$(UpdateVersionProperties)' == '' ">true</UpdateVersionProperties>
 
         <UseFullSemVerForNuGet Condition=" '$(UseFullSemVerForNuGet)' == '' ">true</UseFullSemVerForNuGet>
+
     </PropertyGroup>
 
     <PropertyGroup>

